### PR TITLE
Added smooth scroll in backtotop button

### PR DESCRIPTION
**Pull request by shubhraagarwal**
As discussed in Issue Number #81 , I have added smooth scrolling for the backtotop button. This contribution is made under GS'SOC21, so kindly add the required labels to this PR.

Thank You :)

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -237,7 +237,9 @@ hr {
 .about-content {
     margin-top: 50px;
 }
-
+.backtotop{
```
Comment (harikesh409) [inline css/style.css:240]: There is no element with this `.backtotop` class in the html file, and there is already `scroll-behavior: smooth;` present for `*` property so there is no need for it again.

File: index.html [hunk 2]
```diff
@@ -43,7 +43,7 @@
     </div>
 <!-- header starts -->
     <header class="fix">
-        <video src="video/video.mp4" class="vid-bg" autoplay loop muted></video>
+        <video src="video/video.mp4" class="vid-bg" autoplay loop muted id="top"></video>
```
Comment (harikesh409) [inline index.html:46]: What is the use of adding an id?

**Comment by harikesh409** [comment]
@shubhraagarwal are you working on the changes?

**Comment by ravikr126** [comment]
are you completed with your work @shubhraagarwal ???

**Comment by shubhraagarwal** [comment]
Please assign it to someone else. I won't be able to contribute. I hope you understand.

**Comment by harikesh409** [comment]
The issue is resolved in #131

**Pull request closed without merging**